akka-docs/src/main/paradox/actors.md

```diff
@@ -1,8 +1,11 @@
-# Actors
+# Classic Actors
+
+Akka Classic is the original Actor APIs now superseeded by the new more type safe actor APIs.
```

Comment:
maybe not say superseded since it has a negative connotation 

Suggestion:

Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use the classic APIs. It is also possible to use Akka Typed together with classic actors within the same `ActorSystem`, see <link to coexisting.md>. For new projects we recommend using the new Actor APIs.

btw, also typo in superseeded

Reply:
Sounds good, I'll use that.